app/assets/stylesheets/application.css.scss

```diff
@@ -110,6 +110,30 @@ body {
   background-position: center center;
 }
 
+.application-body-edoras{
+  background-color: black;
+  background-image: url('edoras.jpg');
+  background-size:     cover;
+  background-repeat:   no-repeat;
+  background-position: center center;
+}
+
+.application-body-minas_tirith{
+  background-color: black;
+  background-image: url('minas_tirith.jpg');
+  background-size:     cover;
+  background-repeat:   no-repeat;
+  background-position: center center;
+}
+
+.application-body-mordor{
+  background-color: black;
+  background-image: url('mordor.jpg');
+  background-size:     cover;
```

Comment:
Properties should be ordered background-color, background-image, background-position, background-repeat, background-size<br>Colon after property should be followed by one space